package.json

```diff
@@ -15,9 +15,9 @@
   "devDependencies": {
     "@graphprotocol/graph-cli": "^0.19.0",
     "@graphprotocol/graph-ts": "^0.19.0",
-    "@keep-network/keep-core": "^1.4.1",
-    "@keep-network/keep-ecdsa": "^1.3.1",
-    "@keep-network/tbtc": "^1.1.0",
+    "@keep-network/keep-core": "^1.8.0-ropsten.3",
+    "@keep-network/keep-ecdsa": "^1.7.0-ropsten.2",
+    "@keep-network/tbtc": "^1.1.2-ropsten.3",
```

Comment:
So once we merge this PR to main the package dependencies will be updated from mainnet to ropsten. I prefer this repository to be able to support multiple networks. Not sure yet what would be the best approach though.

Reply:
This change is actually for local development only and when the workflow runs these versions [will be replaced](https://github.com/keep-network/allthekeeps/blob/allthekeeps-on-ropsten/.github/workflows/allthekeeps-testnet.yml#L54-L56). The whole [issue](https://keepnetwork.atlassian.net/browse/CP-39) of ATK PRs was to make it work on Ropsten since the mainnet is supported by the community. But yea, being able to switch is certainly a good option in case we'd want to switch our test network (although I guess that would require much more coordination). I think what we can do without digging too much is to use the same approach as we have in workflow that runs ATK, ie. passing `keep-*` package versions `KEEP_CORE_VERSION`, `KEEP_ECDSA_VERSION` & `KEEP_TBTC_VERSION`. As for now if you don't pass specific package versions, it will just get the latest Ropsten packages (this also answers your [other comment](https://github.com/keep-network/keep-subgraph/pull/1#discussion_r702965373)). I can revert this change, but then it will work as follows: if we pass the versions, then the project will work with those packages; otherwise, it will point to mainnet. wdyt?